scipy/stats/_crosstab.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+
+
+def crosstab(*args, levels=None):
+    """
+    Create a table of the counts of the tuples in ``zip(*args)``.
+
+    When ``len(args) > 1``, the array computed by this function is
+    often referred to as a *contingency table* [1]_.
+
+    The arguments must be sequences with the same length.  The second return
+    value, `count` is an integer array with ``len(args)`` dimensions.  If
+    `levels` is None, the shape of `count` is ``(n0, n1, ...)``, where ``nk``
+    is the number of unique elements in ``args[k]``.
+
+    Parameters
+    ----------
+    args : sequences
+        A sequence of sequences whose unique aligned elements are to be
+        counted.  The sequences in args must all be the same length.
+    levels : sequence, optional
+        If `levels` is given, it must be a sequence that is the same length as
+        `args`.  Each element in `levels` is either a sequence or None.  If it
+        is a sequence, it gives the values in the corresponding sequence in
+        `args` that are to be counted.  If any value in the sequences in `args`
+        does not occur in the corresponding sequence in `levels`, that value
+        is ignored and not counted in the returned array `count`.
+
+    Returns
+    -------
+    elements : tuple of numpy.ndarrays.
+        Tuple of length ``len(args)`` containing the arrays of elements that
+        are counted in `count`.  These can be interpreted as the labels of
+        the corresponding dimensions of `count`.
+    count : numpy.ndarray
+        Counts of the unique elements in ``zip(*args)``, stored in an array.
+        Also known as a *contingency table* when ``len(args) > 1``.
+
+    See Also
+    --------
+    numpy.unique
+
+    Notes
+    -----
+    .. versionadded:: 1.5.0
+
+    References
+    ----------
+    .. [1] "Contingency table", http://en.wikipedia.org/wiki/Contingency_table
+
+    Examples
+    --------
+    >>> from scipy.stats import crosstab
+
+    Given the lists `a` and `x`, create a contingency table that counts the
+    frequencies of the corresponding pairs.
+
+    >>> a = ['A', 'B', 'A', 'A', 'B', 'B', 'A', 'A', 'B', 'B']
+    >>> x = ['X', 'X', 'X', 'Y', 'Z', 'Z', 'Y', 'Y', 'Z', 'Z']
+    >>> (avals, xvals), count = crosstab(a, x)
+    >>> avals.tolist()
```

Comment:
Why `.tolist()`?

Reply:
At the time, I thought the output looked nicer, but if a typical reader of the docstring has the same reaction as you, I shouldn't use it.  I removed it in the update.